Bottom: c6062879c4cbea65cb0bca9c7b4cb03d6c8e87f4
Top:    933af9fa2d01bb141fe78317e1f68558f4a8f4b3
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-10 14:32:48 -0500

Refresh of parameterize-uris

---
diff --git a/docker/buildbot-worker-ntel/Dockerfile b/docker/buildbot-worker-ntel/Dockerfile
index 11a726fb..cd2088e9 100644
--- a/docker/buildbot-worker-ntel/Dockerfile
+++ b/docker/buildbot-worker-ntel/Dockerfile
@@ -67,12 +67,14 @@ RUN wget https://bootstrap.pypa.io/get-pip.py \
     buildbot-worker==${BUILDBOT_VERSION}
 
 # Install software for image signing
-RUN curl http://172.16.64.2/~georgem/610-009981-015_SW_PTK_5.3_Client_RevA.tar \
+ARG PTK_URI="http://172.16.64.2/~georgem/610-009981-015_SW_PTK_5.3_Client_RevA.tar"
+ARG PTK_HSM_HOST="172.16.64.38"
+RUN curl ${PTK_URI} \
 |   tar -v -x -C /tmp \
 &&  alien --to-deb --install --scripts /tmp/610-009981-015_SW_PTK_5.3_Client_RevA/SDKs/Linux64/ptkc_sdk/PTKcpsdk-5.3.0-16.x86_64.rpm \
 &&  alien --to-deb --install --scripts /tmp/610-009981-015_SW_PTK_5.3_Client_RevA/SDKs/Linux64/ptkc_runtime/PTKcprt-5.3.0-15.x86_64.rpm \
 &&  alien --to-deb --install --scripts /tmp/610-009981-015_SW_PTK_5.3_Client_RevA/SDKs/Linux64/network_hsm_access_provider/PTKnethsm-5.3.0-15.x86_64.rpm \
-&&  echo "ET_HSM_NETCLIENT_SERVERLIST=172.16.64.38" > /etc/default/et_hsm \
+&&  echo "ET_HSM_NETCLIENT_SERVERLIST=${PTK_HSM_HOST}" > /etc/default/et_hsm \
 &&  echo "/opt/safenet/protecttoolkit5/ptk/lib" > /etc/ld.so.conf.d/safenet_ptk.conf \
 &&  ldconfig -v \
 &&  rm -rf /tmp/610-009981-015_SW_PTK_5.3_Client_RevA/
